x/concentrated-liquidity/position_test.go

```diff
@@ -2245,3 +2246,298 @@ func (s *KeeperTestSuite) TestNegativeTickRange_SpreadFactor() {
 
 	s.clk.InitGenesis(s.Ctx, *export)
 }
+
+func (s *KeeperTestSuite) TestTransferPositions() {
```

Comment:
Could you add comments about the invariants that this test aims to validate?

Reply:
Comments added https://github.com/osmosis-labs/osmosis/pull/6623/commits/b17bd2c9131aef6d05409c563cd2a900bf5644b3